src/interpreter/index.ts

```diff
@@ -185,6 +185,33 @@ export class Interpreter implements Expr.Visitor<BrsType>, Stmt.Visitor<BrsType>
         }
     }
 
+    callNamedFunction(functionName: string) {
+        try {
+            let callbackVariable = new Expr.Variable({
+                kind: Lexeme.Identifier,
+                text: functionName,
+                isReserved: false,
+                location: {
+                    start: {
+                        line: -1,
+                        column: -1,
+                    },
+                    end: {
+                        line: -1,
+                        column: -1,
+                    },
+                    file: "(internal)",
+                },
+            });
+            let maybeCallback = this.evaluate(callbackVariable);
+            if (maybeCallback.kind === ValueKind.Callable) {
+                this.visitCall(new Expr.Call(callbackVariable, callbackVariable.name, []));
+            }
+        } catch (err) {
+            throw err;
```

Comment:
It feels a little weird to try-catch-throw here without doing using `err` at all — would it be reasonable to remove the `try/catch` entirely?

Reply:
You're right, it doesn't make any sense to just throw the error, gonna remove this.